### ISSUE TITLE
Doesn't find built-in Mac applications

### ISSUE THREAD

**Issue by mysteryDate** [quicksilver/Quicksilver#2516]
I think this began when updating to Catalina. Quicksilver will not be able to find applications like Quicktime, Calendar or iTunes. It works fine for applications I've installed myself (Chrome, etc.). Even if I use Quicksilver to find the Applications folder, then go into it, it still doesn't see any of the built-in apps.
  I've tried re-scanning, restarting and even re-installing Quicksilver but the problem persists.

**SCREENSHOTS**
<img width="869" alt="Screen Shot 2020-03-29 at 7 43 22 AM" src="https://user-images.githubusercontent.com/2529989/77848325-f0100600-7191-11ea-9b0b-9a8ff3bf4347.png">
<img width="453" alt="Screen Shot 2020-03-29 at 7 49 25 AM" src="https://user-images.githubusercontent.com/2529989/77848330-f30af680-7191-11ea-8fd0-92d502104148.png">

**What version of macOS?**
10.15.4 (Catalina)

**What version of Quicksilver?**
16.0

**Comment by skurfer** [comment]
This is fixed by #2510 but a new version with the fix hasn't been released yet.

Until then, you can create a manual catalog entry for `/System/Applications`.

**Comment by mysteryDate** [comment]
Works great! Thanks

**Comment by ernstki** [comment]
Have any of you guys noticed this starting to recur?

After a recent update, built-in macOS apps like Messages weren't showing up in my catalog anymore, so I searched the changelog for "applications" and found this issue. However, attempting to add `/System/Applications` to the catalog manually causes Quicksilver to crash every time (I did send the crash reports).

I use a non-English localization, so `/System/Applications` tab-completes in the <kbd>Shift</kbd>+<kbd>⌘</kbd>+<kbd>G</kbd> dialog as `/System/Programme`, and that's how it shows up in icon views in file pickers and Finder as well. So I dunno if that has something to do with it?

The Quicksilver application already has full disk access in System Preferences. I did double-check that.
